{"uuid":"011d184c-d3a3-4ace-8ba9-ab65b6f9b5f6","historyId":"76c347ddf5f27c890d23b9369ca4fd40","testCaseId":"[engine:junit-jupiter]/[class:org.academo.academo.London.service.SubmissionServiceTest]/[method:getByTaskId_DataAccessException()]","testCaseName":"Get By Task ID - Throws DatabaseServiceException on database error","fullName":"org.academo.academo.London.service.SubmissionServiceTest.getByTaskId_DataAccessException","labels":[{"name":"tag","value":"unit"},{"name":"junit.platform.uniqueid","value":"[engine:junit-jupiter]/[class:org.academo.academo.London.service.SubmissionServiceTest]/[method:getByTaskId_DataAccessException()]"},{"name":"host","value":"Rockio"},{"name":"thread","value":"58516@Rockio.main(3)"},{"name":"framework","value":"junit-platform"},{"name":"language","value":"java"},{"name":"package","value":"org.academo.academo.London.service.SubmissionServiceTest"},{"name":"testClass","value":"org.academo.academo.London.service.SubmissionServiceTest"},{"name":"testMethod","value":"getByTaskId_DataAccessException"},{"name":"suite","value":"Submission Service Tests"}],"links":[],"name":"Get By Task ID - Throws DatabaseServiceException on database error","status":"passed","stage":"finished","description":"","steps":[],"attachments":[],"parameters":[],"start":1757153507209,"stop":1757153507211}